userguide/system.rst

```diff
@@ -147,54 +147,55 @@ can be configured using the General tab:
-   |                      |                | <https://www.redbrick.dcu.ie/~d_fens/articles/Firefox:_This_Address_is_Restricted>`__                                    |
+   |                      |                | <https://www.redbrick.dcu.ie/~d_fens/articles/Firefox:_This_Address_is_Restricted>`__.                                   |
+   |                      |                |                                                                                                                          |
    +----------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
-   | WebGUI HTTPS Port    | integer        | allows configuring a non-standard port for accessing the administrative GUI over HTTPS                                   |
+   | WebGUI HTTPS Port    | integer        | Allow configuring a non-standard port for accessing the administrative GUI over HTTPS.                                   |
    |                      |                |                                                                                                                          |
    +----------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
-   | WebGUI HTTP -->      | checkbox       | when this box is checked, *HTTP* connections are automatically redirected to                                             |
-   | HTTPS Redirect       |                | *HTTPS* if                                                                                                               |
-   |                      |                | *HTTPS* is selected in :guilabel:`Protocol`, otherwise such connections will fail                                        |
+   | WebGUI HTTP -->      | checkbox       | Set to redirect *HTTP* connections to *HTTPS*.                                                                           |
+   | HTTPS Redirect       |                | *HTTPS* must be selected in :guilabel:`Protocol`.                                                                        |
+   |                      |                |                                                                                                                          |
    |                      |                |                                                                                                                          |
    |                      |                |                                                                                                                          |
    +----------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Not a huge issue here but "*HTTPS* must be selected in :guilabel:`Protocol`." could go on the previous line, and 3 of the 4 blank lines can be deleted.